src/shrub/js/grade/grade.js

```diff
@@ -3,6 +3,7 @@ import Fetch	 				from '../internal/fetch';
 export default {
 	GetMy,
 	GetInfo,
```

Comment:
`GetInfo` should be removed (was supposed to be `GetGames`, but I got the name wrong)

Reply:
Removed `GetInfo` (the function too)